CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 | What's new | |
 | :--------- | :-: |
-| Schema Proposals can now pull updates from it's target variant. Proposals that have conflicts with its source variant can go into a stuck state where it cannot be implemented. These can now be resolved by pulling from source variant and resolving the conflicts. | <img width="989" alt="image" src="https://github.com/user-attachments/assets/aaaa4664-f505-47f5-8a3a-18968b5a9b25">
+| Schema Proposals can now pull updates from its target variant. Proposals that have conflicts with its source variant can go into a stuck state where it cannot be implemented. These can now be resolved by pulling from source variant and resolving the conflicts. | <img width="989" alt="image" src="https://github.com/user-attachments/assets/aaaa4664-f505-47f5-8a3a-18968b5a9b25">
```

Comment:
I have a couple of questions for clarification:

1. The first sentence says proposals can pull from target variants, but the last sentence says proposals can pull from source variants--are these two separate capabilities or are we using "target" and "source" interchangably? (Assuming the latter.)
2. Does the second sentence describe the previous state? (Assuming yes.)
3. Do you think we need documentation around this new capability or is the UI obvious enough that it doesn't need documentation?

If my assumptions for 1 and 2 are correct, I would suggest this:

```suggestion
| Schema proposals can now pull updates from their source variants. Previously, conflicts with source variants could cause proposals to enter a stuck state, preventing implementation. You can now address this by pulling changes from source variants and resolving the conflicts. | <img width="989" alt="image" src="https://github.com/user-attachments/assets/aaaa4664-f505-47f5-8a3a-18968b5a9b25">
```

Reply:
1. Yup using them interchangeably 🙏 
2. Proposals with conflicts will still be in a stuck state, can't become implemented until someone pulls and resolves conflicts. I'd say maybe we should drop the `Previoiusly,`?
3. I think it's mostly obvious enough, let's see what feedback we get :)

Reply:
will take the suggesetion but taking the "Previously," off
 
```suggestion
| Schema proposals can now pull updates from their source variants. Conflicts with source variants could cause proposals to enter a stuck state, preventing implementation. You can now address this by pulling changes from source variants and resolving the conflicts. | <img width="989" alt="image" src="https://github.com/user-attachments/assets/aaaa4664-f505-47f5-8a3a-18968b5a9b25">
```

Reply:
Sounds good, thanks @cheapsteak !